Formula/llvm.rb

```diff
@@ -188,17 +226,36 @@ def install
     args = %w[
       -DLLVM_OPTIMIZED_TABLEGEN=On
       -DLLVM_BUILD_LLVM_DYLIB=On
+      -DLLVM_TARGETS_TO_BUILD=X86
```

Comment:
I'm fine with enabling all of them or adding ARM or whatever. I think X86, AMDGPU (for those with Mac Pros), NVPTX (for those with NVIDIA cards) will cover 99% of Homebrew users. One of my colleagues works with ARM chips (not on a OS X though), so I can certainly understand if someone wants it, so I'm OK to include it. I'm also fine with `--with-all-targets` option.